lib/update.js

```diff
@@ -38,11 +39,21 @@ exports.update = function update(callback) {
           callback(err, results);
 
           if (source && !err) {
-            fs.writeFile(exports.output, source, function idk(err) {
+	    tmp.file(function (err, tempFilePath) {
               if (err) {
-                console.error('Failed to save the generated file due to reasons', err);
+                console.error('Failed to create temporary file for regexp.js', err);
+                return;
```

Comment:
Instead of returning, the callback function should be called with the error. Same counts for the other `return` statement in the changes.